rpc/ethereum/namespaces/debug/api.go

```diff
@@ -351,7 +351,10 @@ func (a *API) StartCPUProfile(file string) error {
 		}
 		if err := pprof.StartCPUProfile(f); err != nil {
 			a.logger.Debug("cpu profiling already in use", "error", err.Error())
-			f.Close()
+			if err := f.Close(); err != nil {
+				a.logger.Debug("failed to close cpu profile file")
+				return errors.New("failed to close cpu profile file")
```

Comment:
can you wrap the error?
```suggestion
				return errors.Wrap(err, "failed to close cpu profile file")
```